Cloneable/CloneableGenerator.cs

```diff
@@ -137,9 +226,11 @@ private static INamedTypeSymbol GetClassSymbol(Compilation compilation, Microsof
             return classSymbol;
         }
 
-        private static void InjectCloneableAttribute(GeneratorExecutionContext context)
+        private static void InjectCloneableAttributes(GeneratorExecutionContext context)
         {
             context.AddSource("CloneableAttribute", SourceText.From(cloneableAttributeText, Encoding.UTF8));
+            context.AddSource("ClonePropertyAttribute", SourceText.From(clonePropertyAttributeText, Encoding.UTF8));
```

Comment:
Define constants for attribute names and use them both here and in their class declarations.

Reply:
All done